zeppelin-server/pom.xml

```diff
@@ -43,19 +43,19 @@
       <dependency>
         <groupId>org.scala-lang</groupId>
         <artifactId>scala-library</artifactId>
-        <version>2.10.4</version>
+        <version>2.11.8</version>
```

Comment:
Just tried this branch and printed a class path of interpreter process (without -Pbuild-distr). And my classpath is

```
-cp ::/zeppelin/interpreter/scalding/*:/zeppelin/zeppelin-interpreter/target/lib/*::/zeppelin/conf:/zeppelin/conf:/zeppelin/zeppelin-interpreter/target/classes
```
Could you verify classpath of scalding interpreter process?